board-viewer.html

```diff
@@ -180,10 +179,10 @@
 Polymer({
   is: "board-viewer",
   properties: {
-    /** Whether the zoom control is visible. */
+    /** Whether the zoom operations are available / control is visible. */
     disableZoom: {
       type: Boolean,
-      value: true,
+      value: false,
```

Comment:
Outside of interactive explorer apps (like the main view we have built,) the typical embedding will be as a clipped and zoomed region used for static (non interactive) illustration and call-out documentation, so the default should be to have the interactive elements (zoom, drag) disabled.

Eventually when zoom is enabled, there will be a corresponding on screen zoom bar as well.